config/api.yml

```diff
@@ -688,6 +688,20 @@
       :delete:
       - :name: delete
         :identifier: condition_delete
+  :configuration_scripts:
+    :description: Configuration Scripts
+    :options:
+    - :collection
+    :verbs: *g
+    :klass: ConfigurationScript
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: configuration_script_view
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: configuration_script_view
```

Comment:
Just for my understanding of how we expose API features - what is the purpose of `_view` here?

Reply:
@syncrou 
We have product features which are stored in a single role. We have the ability to read/write features. There is already a feature defined for Ansible Tower Job Templates.
https://github.com/ManageIQ/manageiq/blob/5080b2f18c39eba0a6eb621b004b0469bc4660d5/db/fixtures/miq_product_features.yml#L4753

Reply:
Ah! - I never knew how those bits worked together.